api/src/Repository/StoryboardSectionRepository.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Repository;
+
+use App\Entity\ContentNode\StoryboardSection;
+use App\Entity\User;
+use Doctrine\ORM\QueryBuilder;
+use Doctrine\Persistence\ManagerRegistry;
+
+/**
+ * @method null|StoryboardSection find($id, $lockMode = null, $lockVersion = null)
+ * @method null|StoryboardSection findOneBy(array $criteria, array $orderBy = null)
+ * @method StoryboardSection[]    findAll()
+ * @method StoryboardSection[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
+class StoryboardSectionRepository extends ContentNodeRepository {
```

Comment:
Streng genommen müsste buildContentNodeFilter in einem Trait+Interface sein, da die Section selber kein ContentNode ist

Reply:
Fixed